tools/run_tests/build_python.sh

```diff
@@ -39,6 +39,46 @@ GRPCIO=$ROOT/src/python/grpcio
 GRPCIO_TEST=$ROOT/src/python/grpcio_test
 GRPCIO_HEALTH_CHECKING=$ROOT/src/python/grpcio_health_checking
 
+#
```

Comment:
In languages that aren't Bash when I see stanza-delineating comments like these I suggest to the author that the file has gotten too large and should be broken up. Is that the case here? This stanza is just two functions; why bother with the comments anyway?